kik/messages/keyboard_message.py

```
@@ -14,7 +15,9 @@ def __init__(self, keyboards=None, **kwargs):
 
     def to_json(self):
         output_json = super(KeyboardMessage, self).to_json()
-        if len(self.keyboards) > 0:
-            output_json['keyboards'] = [keyboard.to_json() for keyboard in self.keyboards]
+        non_empty_srs = any([keyboard.responses if hasattr(keyboard, 'responses') else True
```

Maybe its better to just raise an exception when someone tries to create an SR keyboard with empty responses?